src/mono/msbuild/AppleTestRunner/AppleTestRunner.cs

```diff
@@ -62,7 +62,7 @@ public static async Task<int> Main(string[] args)
 
     protected override IEnumerable<TestAssemblyInfo> GetTestAssemblies()
     {
-        foreach (string file in testLibs)
+        foreach (string file in testLibs!)
```

Comment:
Why is the null forgiving operator needed here?

Reply:
@safern I forgot to remove it

Reply:
Will fix it in the next PR